### PR TITLE
Pull in upstream updates and work around unsigned RPM failures

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

In this pull request I:
* Pull in upstream updates so that this repo benefits from "recent" improvements such as [lineage](https://github.com/cisagov/action-lineage)
* Add a workaround for the fact that `dnf`/`rpm`/`yum` cannot verify the signature of the Cloudwatch Agent RPM
* Remove Fedora 33 from the molecule tests since Fedora 33 is currently borked; it cannot even perform a simple `dnf update` because of GPG signature issues with Fedora RPMs

## 💭 Motivation and Context

This change is required because this Ansible role is required to build our COOL AMIs.

## 🧪 Testing

All pre-commit hooks and molecule tests pass. I am in the process of building a new FreeIPA image using this branch and will update this paragraph after testing.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tasks/install_RedHat.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 - name: Install AWS CloudWatch Agent RPM via dnf
   dnf:
+    disable_gpg_check: yes
```
Comment (dav3r) [inline tasks/install_RedHat.yml:6]: We should include a comment here, similar to what you put in https://github.com/cisagov/ansible-role-amazon-ssm-agent/pull/8/commits/07efeb0a5b90e8650ed5ecfca0bdda3ed091d1f5.

**Review by dav3r — APPROVED** [review]
Thanks for bringing this one up to date! ☘️ 
Please note my one small RFC.

File: tasks/install_RedHat.yml (hunk 1 above)
Comment (jsf9k) [reply tasks/install_RedHat.yml:6]: See commit 3c0461a.

**Review by mcdonnnj — APPROVED** [review]
Looks Mighty Good to Me. 💪

**Pull request merged**